tests/tests.rs

```diff
@@ -31,20 +31,27 @@ fn compare_generated_header(header: &PathBuf,
     if !expected.is_file() {
         let file_name = expected.file_name().unwrap().to_owned();
         expected.pop();
-
-        if cfg!(feature = "testing_only_libclang_4") {
-            expected.push("libclang-4");
-        } else if cfg!(feature = "testing_only_libclang_3_9") {
-            expected.push("libclang-3.9");
-        } else if cfg!(feature = "testing_only_libclang_3_8") {
-            expected.push("libclang-3.8");
+        
+        let version = clang_version();
+        match version.parsed {
+            None => panic!("Couldn't parse libclang version!"),
+            Some(version) => {
+                let (maj, min) = version;
+                if maj == 4 {
```

Comment:
I use libclang 5.0 from trunk, so this should probably be `if maj >= 4`, to handle that correctly (right now there aren't differences between libclang 4 y 5 regarding tests)